src/chains/ethereum/transaction/src/eip2930-access-list-transaction.ts

```diff
@@ -242,4 +240,6 @@ export class EIP2930AccessListTransaction extends RuntimeTransaction {
       chainId
     );
   }
+
+  public updateEffectiveGasPrice() {}
```

Comment:
Definitely open to suggestions on making this cleaner or omitting this method. From what I understand about the abstract method that calls this, we have to have this method defined.